### PR TITLE
Add mixin template to add a `serialize` method to a container

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
(As featured in https://github.com/sociomantic-tsunami/dhtproto/pull/110.)

File: src/swarm/neo/client/mixins/DeserializeMethod.d [hunk 1]
```diff
@@ -74,3 +75,37 @@ template DeserializeMethod ( alias src )
...
 public VersionDecorator client_deserializer_version_decorator;
+
+version ( UnitTest )
+{
+    import ocean.core.Test;
+    import ocean.util.serialize.contiguous.Contiguous;
+    import ocean.util.serialize.contiguous.Serializer;
+
+   struct Request
+    {
```
Comment (nemanja-boric-sociomantic) [inline src/swarm/neo/client/mixins/DeserializeMethod.d]: nitpick: spacing

**Review by nemanja-boric-sociomantic — APPROVED** [review]

File: src/swarm/neo/client/mixins/DeserializeMethod.d (hunk 1 above)
Comment (gavin-norman-sociomantic) [reply src/swarm/neo/client/mixins/DeserializeMethod.d]: Oh yeah. That is weird.

Comment (gavin-norman-sociomantic) [reply src/swarm/neo/client/mixins/DeserializeMethod.d]: Updated. (And rebased.)

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Comment by nemanja-boric-sociomantic** [comment]
Approved again.

**Pull request merged**